lib/groups/BasicGroup.js

```diff
@@ -1,9 +1,20 @@
-const GroupHelper = require('../utils/group-helper');
+const GroupHelper = require('../utils/GroupHelper');
+const { core, groups } = require('../utils/cli-flags');
 
 class BasicGroup extends GroupHelper {
     constructor(options) {
         super(options);
-        this.WEBPACK_OPTION_FLAGS = ['prod', 'dev', 'watch', 'w', 'prod', 'p', 'interactive', 'i', 'defaults', 'progress'];
+        this.WEBPACK_OPTION_FLAGS = core
```

Comment:
Reading from `cli-flags` is better because we keep everything in one single place

Reply:
May leverage `core-flags.js` here to avoid redundant filter and reduce.

Reply:
Here we do different transformation. I don't think we need this here. I just kept it to avoid possible regressions

Reply:
Yeah, they are different